rwkv.cpp

```diff
@@ -499,7 +523,11 @@ struct rwkv_context * rwkv_init_from_file(const char * file_path, const uint32_t
 
     while (true) {
         int32_t dim_count, key_length, data_type;
-        RWKV_ASSERT_NULL_MSG(RWKV_ERROR_MODEL_PARAMS | RWKV_ERROR_FILE_READ, fread(&dim_count, sizeof(int32_t), 1, file) == 1 || feof(file), "Failed to read an int32 value from a file (dim_count)");
+        RWKV_ASSERT_NULL_MSG(
+            RWKV_ERROR_MODEL_PARAMS | RWKV_ERROR_FILE_READ,
+            fread(&dim_count, sizeof(int32_t), 1, file) == 1 || feof(file),
+            "Failed to read an int32 value from a file (dim_count)"
+        );
         if (feof(file)) break;
```

Comment:
forgot to add braces

```suggestion
        if (feof(file)) { break; }
```